### DISCUSSION TITLE
C# Source Generators: How to get build information?

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#83] [General]
# C# Source Generators: How to get build information?

Source generators are a powerful feature introduced to C#, allowing developers to generate additional code during the compilation process automatically. They can help reduce boilerplate, improve performance, and simplify your codebase.
This blog post will introduce source generators, discuss how they work, and walk through an example of a source generator for generating build information.

https://steven-giesel.com/blogPost/cec8df6e-b271-4b4c-8ff6-e9f3aa5e26a1

**Comment by lindexi** [comment]
Thank you
